README.md

```diff
@@ -388,7 +388,7 @@ assert config == d
   - Scott Johnson ([@scottj97](https://github.com/scottj97))
   - Yifei Tao ([@yifeitao](https://github.com/yifeitao))
   - Kevin Fong ([@KevinMFong](https://github.com/KevinMFong))
-  - Erik Cederstrand [@ecederstrand](https://github.com/ecederstrand))
+  - Erik Cederstrand ([@ecederstrand](https://github.com/ecederstrand))
```

Comment:
This isn't precisely a spelling error, it's a stylistic error that turned up as I was tuning check-spelling to ignore the items in this list.